sdk/test-utils/recorder/package.json

```diff
@@ -99,6 +98,46 @@
     "rimraf": "^3.0.0",
     "tslib": "^2.2.0",
     "ts-node": "^10.0.0",
-    "typescript": "~5.3.3"
-  }
+    "tshy": "1.11.1",
+    "typescript": "~5.3.3",
+    "vitest": "^1.0.0",
```

Comment:
Why are we using vitest at that version and browser/coverage to differ?

Reply:
Updated in a later commit, just couldn't push, something wrong with codespaces auth.